homework_7/task_1.c

```diff
@@ -0,0 +1,126 @@
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+void printInFile(FILE* file, int* array)
+{
+    fprintf(file, "%d %d\n", array[0], array[1]);
+    for (int i = 0; i < array[0] + 1; i++)
+        fprintf(file, "%d ", array[i + 2]);
+}
+
+void findShortestPath(int nVertex, int** matrix, int start, int end, FILE* file)
+{
+    int* lengths = calloc(nVertex, sizeof(int));
+    bool* used = calloc(nVertex, sizeof(bool));
+    int* parents = calloc(nVertex, sizeof(int));
+    for (int i = 0; i < nVertex; i++)
+        parents[i] = -1;
+    for (int i = 0; i < nVertex; i++) {
+        lengths[i] = 1000000;
```

Comment:
Hard-coding values is a bad practice. Please use additional variable for it  across the code